finisher_api.go

```diff
@@ -173,7 +173,7 @@ func (db *DB) Find(dest interface{}, conds ...interface{}) (tx *DB) {
 // FindInBatches find records in batches
 func (db *DB) FindInBatches(dest interface{}, batchSize int, fc func(tx *DB, batch int) error) *DB {
 	var (
-		tx = db.Order(clause.OrderByColumn{
+		tx = db.Session(&Session{}).Order(clause.OrderByColumn{
```

Comment:
test failed caused by here, modify it can be successful
```go
tx           = db.Session(&Session{})
```

Reply:
https://www.postgresql.org/docs/current/queries-limit.html
 > it is an inherent consequence of the fact that SQL does not promise to deliver the results of a query in any particular order unless ORDER BY is used to constrain the order.